selftests/unit/runner_tap.py

```diff
@@ -0,0 +1,116 @@
+import os
```

Comment:
Since the tap runner code is in `avocado.plugins.runners` shouldn't these tests be in `selftests.unit.plugin.runners`?

Reply:
Yes, absolutely! Good catch!